mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -490,6 +490,10 @@ void CMomentumPlayer::Spawn()
             ClearStartMark(i, false);
         }
 
+        // Load startmarks after player spawn
+        if (this == GetLocalPlayer())
```

Comment:
There's only the local player. This check isn't needed.

Reply:
Forgot about that, thanks. This check is now removed.